src/components/Books.js

```diff
@@ -0,0 +1,60 @@
+const Books = () => {
+  const books = [{ id: 1 }, { id: 2 }, { id: 3 }];
+
+  return (
+    <>
+      <ul className="list-unstyled row mx-0 g-0 gy-3">
+        {books.map((book) => (
+          <li key={book.id} className="card">
+            <div className="row row-cols-auto mx-0 py-4 justify-content-around">
+              <div>
+                <p>Action</p>
+                <p>The Hunger Games</p>
+                <p>Suzanne Collins</p>
+                <div className="row row-cols-auto mx-0 g-0">
+                  <button type="button">Comments</button>
+                  <div className="v-line px-0 mx-3" />
+                  <button type="button">Remove</button>
+                  <div className="v-line px-0 mx-3" />
+                  <button type="button">Edit</button>
+                </div>
+              </div>
+              <div>
+                <div className="row row-cols-auto mx-0">
+                  <div className="Oval-2" />
+                  <div>
+                    <p>64%</p>
+                    <p>Completed</p>
```

Comment:
- [ ] You should have a BookList component that renders a list of books using the Book component, so this means you need to create both the BookList component and the Book component. Separate this file to enable these changes. Remember that the Book component should take the book data as a prop and renders the title and the author.